tests/fixtures/saas/stripe_fixtures.py

```diff
@@ -292,7 +291,7 @@ def stripe_create_erasure_data(
                 "card": {
                     "number": 4242424242424242,
                     "exp_month": 4,
-                    "exp_year": 2023,
+                    "exp_year": datetime.today().year + 1,
```

Comment:
Making sure the test data always has a future expiration date

Reply:
Nice, another test resolved 💥